pipenv/core.py

```diff
@@ -517,7 +517,7 @@ def ensure_project(
     """Ensures both Pipfile and virtualenv exist for the project."""
 
     # Automatically use an activated virtualenv.
-    if project.s.PIPENV_USE_SYSTEM:
+    if project.s.PIPENV_USE_SYSTEM or project.virtualenv_exists:
         system = True
     if not project.pipfile_exists and deploy:
```

Comment:
To avoid confusion down the road, I suggest renaming variable `system` to `system_or_exists`.

Reply:
`system` quite often exists in the code and within many functions. Shall I replace it in everyone with `system_or_exists`?

Reply:
No, just this one. Because that is what it determines here.